sklearn/metrics/_plot/regression.py

```diff
@@ -0,0 +1,454 @@
+    >>> from sklearn.datasets import load_diabetes
+    >>> from sklearn.linear_model import Ridge
+    >>> from sklearn.metrics import PredictionErrorDisplay
+    >>> X, y = load_diabetes(return_X_y=True)
+    >>> ridge = Ridge().fit(X, y)
+    >>> y_pred = ridge.predict(X)
+    >>> display = PredictionErrorDisplay(y_true=y, y_pred=y_pred)
+    >>> display.plot()
+    <...>
+    >>> plt.show()
+    """
+
+    def __init__(
+        self,
+        *,
+        y_true,
+        y_pred,
+        scores=None,
+    ):
+        self.y_true = y_true
+        self.y_pred = y_pred
+        self.scores = scores
+
+    def plot(
+        self,
+        ax=None,
+        *,
+        kind="predictions",
+        scatter_kwargs=None,
+        line_kwargs=None,
+        errors_kwargs=None,
+    ):
+        """Plot visualization.
+
+        Extra keyword arguments will be passed to matplotlib's ``plot``.
+
+        Parameters
+        ----------
+        ax : matplotlib axes, default=None
+            Axes object to plot on. If `None`, a new figure and axes is
+            created.
+
+        kind : {"predictions", "predictions_residuals", "residuals"}, \
+                default="predictions"
+            The type of plot to draw:
+
+            - "predictions" draws the predicted values vs. the true values.
+            - "predictions_residuals" is similar to "predictions" but also
+              shows the errors between the predictions and the true values.
+            - "residuals" draws the residuals vs. the predicted values.
+
+        scatter_kwargs : dict, default=None
+            Dictionary with keywords passed to the `matplotlib.pyplot.scatter`
+            call.
+
+        line_kwargs : dict, default=None
+            Dictionary with keyword passed to the `matplotlib.pyplot.plot`
+            call to draw the diagonal line.
+
+        errors_kwargs : dict, default=None
+            Dictionary with keyword passed to the `matplotlib.pyplot.plot`
+            call to draw the residual lines. Only taken into account when
+            `with_errors=True`.
+
+        Returns
+        -------
+        display : :class:`~sklearn.metrics.plot.PredictionErrorDisplay`
+            Object that stores computed values.
+        """
+        check_matplotlib_support("PredictionErrorDisplay.plot")
+
+        expected_kind = ("predictions", "predictions_residuals", "residuals")
+        if kind not in expected_kind:
+            raise ValueError(
+                f"`kind` must be one of {', '.join(expected_kind)}. "
+                f"Got {kind!r} instead."
+            )
+
+        import matplotlib.pyplot as plt
+
+        if scatter_kwargs is None:
+            scatter_kwargs = {}
+        if line_kwargs is None:
+            line_kwargs = {}
+        if errors_kwargs is None:
+            errors_kwargs = {}
+
+        default_scatter_kwargs = {"color": "black", "alpha": 0.8}
+        default_line_kwargs = {
+            "color": "tab:blue",
```

Comment:
I find that having the line in blue and the dots in black make the plot a bit sad. What do you think about switching and plot the dots in blue and the line in black (or red) ?